### PR TITLE
Cleanup requirements.txt

### PR DESCRIPTION
**Pull request by elazarg**
enum34 is not needed, and can interfere with gdown
other packages are not shown by `pip freeze`
Also: bump beautifulsoup4 to 4.8.0 - the latest we're compatible with

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -51,4 +50,4 @@ urllib3==1.25.9
 webassets==2.0
 xlrd==1.2.0
 yuicompressor==2.4.8
-lxml==4.5.1
\ No newline at end of file
+lxml==4.5.1
```
Comment (elazarg) [inline requirements.txt:53]: I have no idea what this is

Comment (atalyaalon) [reply requirements.txt:53]: @elazarg it's a whitespace change.
Perhaps windows' new line?
Screen Shot 2020-05-25 at 17 41 50

Comment (elazarg) [reply requirements.txt:53]: I've made the change through the web interface

**Comment by atalyaalon** [comment]
@elazarg please resolve conflicts (due to another merge)
Thanks!!

**Comment by isavion** [comment]
Really important work!
I think that in the future it is better to squash those commits before merging,
so any commit on dev (and later on master) will be a working version.

**Comment by elazarg** [comment]
I agree.

**Pull request merged**